.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
```

Comment:
Thought the trial TravisCI account expired and different directions were being explored elsewhere

Reply:
Hmm. I'm not aware of any of this. I'm not sure if devx was even using the trial TravisCI account, we may have had our own as a legacy of when we were in product

Reply:
We could also be using a free account, we only use travis for public repos as far as I know

Reply:
Thought that might be the case but checking.

Reply:
👍